example/usage_dataset.ipynb

```diff
@@ -219,6 +219,37 @@
     "total_df.loc[:, total_df.columns.str.contains(\"per\")].describe().T"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can show the specific kind of complements that were applied to the records of each country with `JHUData.show_complement()` method."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The possible kinds of complement for each country are the following: \"Monotonic_confirmed\", \"Monotonic_fatal\", \"Monotonic_recovered\", \"Full_recovered\" and \"Partial_recovered\"."
```

Comment:
Could you add desctriptive names and summaries of complement solutions, like this (could be changed)?
```
1. "Monotonic_*": monotonic increasing complment
Force the variable show monotonic increasing.

2. "Full_recovered": fully complement of recovered data
Estimate the number of recovered cases with the value of recovery period.

3. "Partial_recovered": partial complement of recovered data
When recovered values are not updated for some days, extrapolate the values.
```